app/views/categories/show.html.erb

```diff
@@ -0,0 +1,69 @@
+<% content_for :styles do %> 
+
+.new-category{
+    display: inline-block;
+    padding: 30px 0; 
+    background-color: green; 
+    color: white;
+    border-radius: 0;
+    font-weight: bold; cursor: 
+    pointer; transition:
+    background-color 0.3s ease;
+    width: 100%;
+    text-align: center;
+    position: fixed;
+    bottom: 0;
+  }
+  .new-category-button {
+	padding: 10px 40px;
+	background-color: green;
+	color: white;
+	text-decoration: none;
+	border: 2px solid white; /* Add a white border */
+	border-radius: 5px;
+	font-weight: bold;
+	cursor: pointer;
+	transition: background-color 0.3s ease;
+	text-align: center;
+  }
+     
+<% end %>
+<div class="transactions-container">
+  <h1 class="category-header">
+    <% if @category.icon.present? %>
+      <%= image_tag @category.icon, alt: "Category icon" %>
+    <% end %>
+    <%= @category.name %>
+  </h1>
+  
+  <ul>
+    <% @transactions.each do |transaction| %>
+      <li class="transaction-item">
+        <div>
+          <%= transaction.name %> - Amount: <%= number_to_currency(transaction.amount) %>
+        </div>
+		<div>
+		Transaction ID: <%= transaction.id %>
+	  </div>
+        <div class="transaction-date">
+          Transaction Date: <%= transaction.created_at %>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+<div class="total-amount">
+  <div class="total">
+  <p>Total Amount:
+  </div>
+  <div class="amount">
+   <%= number_to_currency(@total_amount) %></p>
+</div>
+</div>
+</div>
+
+<div class="new-category">
+  <%= link_to 'Add a new transaction', new_category_transaction_path(@category), class: 'new-category-button' %>
+</div>
+
+</div>
+</div>
```

Comment:
- [ ] [OPTIONAL] - While this seems to have been missed by the last reviewer @zieeco , in the category show page, the back icon should have been placed at the navbar (At the top left and not at the bottom)

**This is how your design looked like:**

![image](https://github.com/Gardimy/Budget_app/assets/86318284/821f09b7-9592-4d29-b6f8-e734e7c5dc65)

**This is where it should have been placed:**

![image](https://github.com/Gardimy/Budget_app/assets/86318284/393a6111-c7c3-48c9-bdad-4b15fdd2cb1e)